### ISSUE TITLE
check_c_api tests fail

### ISSUE THREAD

**Issue by agoscinski** [Luthaf/rascaline#12]
```
---- check_c_api stdout ----
thread 'check_c_api' panicked at 'failed to run tests: Os { code: 2, kind: NotFound, message: "No such file or directory" }', rascaline-c-api/tests/c-api.rs:46:33
stack backtrace:
   0: rust_begin_unwind
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/panicking.rs:92:14
   2: core::option::expect_none_failed
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/option.rs:1268:5
   3: core::result::Result<T,E>::expect
             at /home/alexgo/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:933:23
   4: c_api::check_c_api
             at ./tests/c-api.rs:46:18
   5: c_api::check_c_api::{{closure}}
             at ./tests/c-api.rs:6:1
   6: core::ops::function::FnOnce::call_once
             at /home/alexgo/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
   7: core::ops::function::FnOnce::call_once
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

Here the full log

```
---- check_c_api stdout ----
thread 'check_c_api' panicked at 'failed to run tests: Os { code: 2, kind: NotFound, message: "No such file or directory" }', rascaline-c-api/tests/c-api.rs:46:33
stack backtrace:
   0:     0x564eea814240 - std::backtrace_rs::backtrace::libunwind::trace::h25e12e0d899beba0
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/../../backtrace/src/backtrace/libunwind.rs:90:5
   1:     0x564eea814240 - std::backtrace_rs::backtrace::trace_unsynchronized::h70e61195d6ae3df6
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x564eea814240 - std::sys_common::backtrace::_print_fmt::hba93ab80d779695a
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/sys_common/backtrace.rs:67:5
   3:     0x564eea814240 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hf092b5883b4b2e50
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/sys_common/backtrace.rs:46:22
   4:     0x564eea83465c - core::fmt::write::hf68bc350a8f2f0dc
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/fmt/mod.rs:1078:17
   5:     0x564eea8104b2 - std::io::Write::write_fmt::hb40620ea640effcf
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/io/mod.rs:1517:15
   6:     0x564eea8164e5 - std::sys_common::backtrace::_print::hd425a11bfe1f20f8
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/sys_common/backtrace.rs:49:5
   7:     0x564eea8164e5 - std::sys_common::backtrace::print::h6d678795c1e61e13
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/sys_common/backtrace.rs:36:9
   8:     0x564eea8164e5 - std::panicking::default_hook::{{closure}}::h78a02a4a0dee5e7e
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:208:50
   9:     0x564eea816072 - std::panicking::default_hook::h56eb7eda02f355a7
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:222:9
  10:     0x564eea816c81 - std::panicking::rust_panic_with_hook::hb27ea14285131c61
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:591:17
  11:     0x564eea8167c7 - std::panicking::begin_panic_handler::{{closure}}::hc552fcee62aad17f
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:497:13
  12:     0x564eea8146fc - std::sys_common::backtrace::__rust_end_short_backtrace::hb9f0aa9a78e885a0
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/sys_common/backtrace.rs:141:18
  13:     0x564eea816729 - rust_begin_unwind
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:493:5
  14:     0x564eea833011 - core::panicking::panic_fmt::h12ac4570ea43d06f
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/panicking.rs:92:14
  15:     0x564eea832e33 - core::option::expect_none_failed::h096fa60f757b7204
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/option.rs:1268:5
  16:     0x564eea7b2779 - core::result::Result<T,E>::expect::h0fd88eb2da814791
                               at /home/alexgo/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:933:23
  17:     0x564eea7b3400 - c_api::check_c_api::h2395b0ea35f202b4
                               at /home/alexgo/lib/rascaline-fork/rascaline-c-api/tests/c-api.rs:46:18
  18:     0x564eea7b1a8a - c_api::check_c_api::{{closure}}::h37c319f1397db0aa
                               at /home/alexgo/lib/rascaline-fork/rascaline-c-api/tests/c-api.rs:6:1
  19:     0x564eea7b382e - core::ops::function::FnOnce::call_once::he19f165f42cb02ac
                               at /home/alexgo/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
  20:     0x564eea7de453 - core::ops::function::FnOnce::call_once::h0fd093fad8ca93e7
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/ops/function.rs:227:5
  21:     0x564eea7de453 - test::__rust_begin_short_backtrace::hcef746a82d49ef5d
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/test/src/lib.rs:516:5
  22:     0x564eea7dcab1 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::h1cd4f92a3ff7b6dd
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/alloc/src/boxed.rs:1328:9
  23:     0x564eea7dcab1 - <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once::hc7727f017980baf2
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panic.rs:322:9
  24:     0x564eea7dcab1 - std::panicking::try::do_call::hf209bd579b8aa20f
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:379:40
  25:     0x564eea7dcab1 - std::panicking::try::h8ddc17bc0112be15
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:343:19
  26:     0x564eea7dcab1 - std::panic::catch_unwind::hcdf0f58b999df7db
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panic.rs:396:14
  27:     0x564eea7dcab1 - test::run_test_in_process::hbf6e8395de708698
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/test/src/lib.rs:538:18
  28:     0x564eea7dcab1 - test::run_test::run_test_inner::{{closure}}::h0e231c614b7eb493
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/test/src/lib.rs:449:39
  29:     0x564eea7b5426 - std::sys_common::backtrace::__rust_begin_short_backtrace::hd111df91fe657985
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/sys_common/backtrace.rs:125:18
  30:     0x564eea7b9fad - std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}::h60f7430ba42b1255
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/thread/mod.rs:474:17
  31:     0x564eea7b9fad - <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once::hde7b6fb3b495e32e
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panic.rs:322:9
  32:     0x564eea7b9fad - std::panicking::try::do_call::h48e91e77a1f18f3e
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:379:40
  33:     0x564eea7b9fad - std::panicking::try::h421a154d71f6a3cf
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:343:19
  34:     0x564eea7b9fad - std::panic::catch_unwind::ha1c5ebfde5253e40
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panic.rs:396:14
  35:     0x564eea7b9fad - std::thread::Builder::spawn_unchecked::{{closure}}::h3b9014913fae8c4d
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/thread/mod.rs:473:30
  36:     0x564eea7b9fad - core::ops::function::FnOnce::call_once{{vtable.shim}}::h944b156da1ee202d
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/ops/function.rs:227:5
  37:     0x564eea81cc1a - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::h9ed215ba67984d70
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/alloc/src/boxed.rs:1328:9
  38:     0x564eea81cc1a - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::hcece06e1fe04906f
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/alloc/src/boxed.rs:1328:9
  39:     0x564eea81cc1a - std::sys::unix::thread::Thread::new::thread_start::h6e82a4b7be15319a
                               at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/sys/unix/thread.rs:71:17
  40:     0x7f686bda06db - start_thread
  41:     0x7f686b52771f - __clone
  42:                0x0 - <unknown>
```

**Comment by Luthaf** [comment]
Which directory are you running `cargo test` from? `/path/to/rascaline/repo` or `/path/to/rascaline/repo/rascaline-c-api`?

**Comment by Luthaf** [comment]
From the backtrace, it looks like `ctest` is not found: https://github.com/Luthaf/rascaline/blob/f108354a03386bd7191699bf385af0472807c369/rascaline-c-api/tests/c-api.rs#L46. Do you have it installed and accessible in PATH? 

I'll try to add some more checks when looking for cmake/ctest to give better error messages.

**Comment by Luthaf** [comment]
After talking with @agoscinski the issue was that ctest was not installed on his machine. This should be fixed by #13.
